modules/core/src/v2/wallet.ts

```diff
@@ -1097,6 +1102,32 @@ export class Wallet {
     }
     this._wallet = await this.bitgo.put(this.url()).send(forwarderFlags).result();
   }
+
+  /**
+     * To manually deploy an ETH address
+     * @param DeployForwardersOptions {Object} - {
+        "id": {String},      
+        "address": {String},
+       }
+     }
+     */
+  async deployForwarders(params: DeployForwardersOptions): Promise<any> {
+    if (_.isUndefined(params.address) && _.isUndefined(params.id)) {
+      throw new Error('address or id of address required');
+    }
+    let query;
+    if (params.address) {
+      query = params.address;
+    } else {
+      query = params.id;
+    }
+    this._wallet = await this.bitgo
+      .post(this.url(`/address/${encodeURIComponent(query)}/deployment`))
```

Comment:
you can maybe set this.url() outside of this call? nested function calls always harder to read